python/tvm/dlight/testing/modules.py

```diff
@@ -0,0 +1,2621 @@
+from tvm.script import ir as I
+from tvm.script import tir as T
+
+# fmt: off
+
+@I.ir_module
+class Decode:
+
```

Comment:
let us not have testing here, instead move to `tests/python/dlight/` so we can grow test cases organically

Reply:
got it

Reply:
moved to `tests/python/dlight/test_modules.py`